cmc-domain-model/src/main/java/uk/gov/hmcts/cmc/claimstore/models/ResponseData.java

```diff
@@ -52,7 +52,6 @@
     @Size(max = 99000)
     private final String defence;
 
-    @NotNull
     @JsonUnwrapped
     private final FreeMediationOption freeMediation;
```

Comment:
Because FreeMediation is nullable we need to make getter return `Optional< FreeMediationOption >` so we avoid confusion later one when somebody tries to use it.

Reply:
yep, I'm working on it. It breaks quite a lot of tests. In about 10 minutes I should push my code